zebra-utils/src/bin/zebra-checkpoints/args.rs

```diff
@@ -2,11 +2,19 @@ use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 pub struct Args {
+    /// Path to zcash-cli command
+    #[structopt(short, long)]
+    pub cli: String,
+
+    /// Use the mainnet network
+    #[structopt(short, long)]
+    pub mainnet: bool,
+
+    /// Use the regtest network
+    #[structopt(short, long)]
+    pub regtest: bool,
+
     /// Use the test network
     #[structopt(short, long)]
     pub testnet: bool,
-
-    /// Passthrough args for `zcash-cli`
-    #[structopt(last = true)]
-    pub zcli_args: Vec<String>,
```

Comment:
We need to support arguments like the zcashd directory, and other custom arguments.

So the passthrough feature is required.

Reply:
Fixed at https://github.com/ZcashFoundation/zebra/pull/740/commits/4655f8192cfe1b6d8cc24bf9da15ae25a1180005